recipes/kart/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "2.5.2" %}
+
+package:
+  name: kart
+  version: {{ version }}
+
+source:
+  url: https://github.com/hsinnan75/Kart/archive/{{ version }}.tar.gz
+  sha256: '99f28beb55ebd118c7e7e27507805d9725f0e490557107cf7058351dbed05981'
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+  host:
```

Comment:
I saw that this builds `htslib` in the `makefile`.
If you want to, you could reuse our pre-built `htslib` for this (also includes `htslib.a`).
Since you statically link, I guess it does not make much difference, but I'm not expert on this.

Reply:
@mbargull i think it may be a custommodified version of `htslib` but i"ll wait for @hsinnan75 to confirm

Reply:
@mbargull The htslib is a modified version. Thanks to @tseemann for the reminder.

Reply:
Good to know, thanks @tseemann!